ethereum/core/src/main/java/org/hyperledger/besu/ethereum/vm/MessageFrame.java

```diff
@@ -1023,12 +1079,18 @@ public void reset() {
     private Boolean isPersistingPrivateState = false;
     private Hash transactionHash;
     private Optional<Bytes> reason = Optional.empty();
+    private ReturnStack returnStack = new ReturnStack();
```

Comment:
Why do we want a 0 capacity return stack as the default?

Reply:
I just changed with the default size of 1023 as in the EIP